source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -119,6 +124,118 @@ def _extend_pt_sections_with_fallback_sections(pt_journey, dp_journey):
         pt_journey.sections.extend(dp_journey.journeys[0].sections)
 
 
+def _make_beginning_car_park_sections(
+    car_park_section,
+    car_park_to_sp_section,
+    dp_extremity,
+    extremity_date_time,
+    pt_journey_extremity,
+    car_park,
+    car_park_crowfly_duration,
+):
+    # car_park section
+    car_park_section.type = response_pb2.SectionType.PARK
+    car_park_section.origin.CopyFrom(dp_extremity)
+    car_park_section.destination.CopyFrom(car_park)
+    car_park_section.begin_date_time = extremity_date_time
+    car_park_section.end_date_time = car_park_section.begin_date_time + CAR_PARK_DURATION
+    car_park_section.duration = CAR_PARK_DURATION
+
+    # crowfly_section
+    car_park_to_sp_section.type = response_pb2.SectionType.CROW_FLY
+    car_park_to_sp_section.street_network.CopyFrom(
+        response_pb2.StreetNetwork(duration=car_park_crowfly_duration, mode=response_pb2.Walking)
+    )
+    car_park_to_sp_section.origin.CopyFrom(car_park)
+    car_park_to_sp_section.destination.CopyFrom(pt_journey_extremity)
+    car_park_to_sp_section.begin_date_time = car_park_section.end_date_time
+    car_park_to_sp_section.end_date_time = car_park_to_sp_section.begin_date_time + car_park_crowfly_duration
+    car_park_to_sp_section.duration = car_park_crowfly_duration
+
+
+def _make_ending_car_park_sections(
```

Comment:
Is `_make_ending_car_park_sections` and `_make_beginning_car_park_sections` are the same fucntion ?
Can you mutualize it ?

Reply:
haha, I tried really hard to mutualize it, yes, I can mutualize them, I will be unreadable